common/lib/xmodule/xmodule/graders.py

```diff
@@ -395,11 +395,11 @@ def grade(self, grade_sheet, generate_random_scores=False):
                 )
             else:
                 percentage = 0.0
-                summary = u"{section_type} {index} Unreleased - 0% (?/?)".format(
+                # Translator's example: Homework 1 - Unreleased - 0% (?/?)
+                summary = _(u"{section_type} {index} Unreleased - 0% (?/?)").format(
                     index=i + self.starting_index,
                     section_type=self.section_type
                 )
-
             short_label = u"{short_label} {index:02d}".format(
```

Comment:
Why not translate this too?

Reply:
Nothing useful to translate here IMO. I'm also unsure what the translator note would look like since `short label` is a variable.

Reply:
Good point.